misc/dex-completion.bash

```diff
@@ -0,0 +1,26 @@
+#/usr/bin/env bash
+
+_get_dx_files()
```

Comment:
This seems unused

Reply:
yep, it was there in case we wanted to move to the second option, removed it!